### ISSUE TITLE
erased_label in evaluation

### ISSUE THREAD

**Issue by lcit** [anilbatra2185/road_connectivity#15]
Hello,
I had a look at the evaluation function for the train_refine_pre.py script at line 260:
https://github.com/anilbatra2185/road_connectivity/blob/4559e3b5922f3a219acbd43d95485ec88ca5e9b4/train_refine_pre.py#L269

could you please explain what the variable `erased_label` contains?

Thanks

**Comment by anilbatra2185** [comment]
Hi @lcit, The "erased_label" variable will hold the manually corrupted road mask with linear type corruption i.e. we add or delete linear lines to road masks. If you view the [notebook](https://github.com/anilbatra2185/road_connectivity/blob/master/visualize_dataset_corrupt.ipynb)  then in cell 7, the last column shows the erased label. Please let me know, if it make sense to you.

Thanks

**Comment by lcit** [comment]
Is the "road mask" the ground-truth? Because it seems to me that you are feeding a corrupted version of the groundtruth (erased_label) into the network and then evaluate the prediction. This biases the results isn't?

**Comment by anilbatra2185** [comment]
sorry for the confusion.

The "erased_label" variable will hold the manually corrupted ground truth road mask with linear type corruption. This is a pre-training stage where we assume that the manual corruption is close to segmentation predictions. In the final stage we transfer the model and fed image along with segmentation prediction to further improve the road connectivity. Please refer to section 3.2 and figure 2 and 4.
